src/test/java/ua_parser/ParserTest.java

```diff
@@ -81,7 +81,7 @@ public void testParseAll() {
 
     Client expected1 = new Client(new UserAgent("Firefox", "3", "5", "5"),
                                   new OS("Mac OS X", "10", "4", null, null),
-                                  new Device("Other"));
+                                  Device.OTHER);
     Client expected2 = new Client(new UserAgent("Mobile Safari", "5", "1", null),
```

Comment:
This has a conflict w another PR that was just merged. once you rebase this change wont be necessary